esp/src/Login.html

```diff
@@ -112,6 +112,10 @@
             function (ready, nlsHPCCMod) {
                 var nlsHPCC = nlsHPCCMod.default;
                 ready(function () {
+                    if (window.location.hash) {
+                        window.localStorage.setItem("redirectAfterLogin", window.location.hash);
+                    }
```

Comment:
If the hash is empty should we remove any existing redirectAfterLogin (I am thinking of the case where a user fails to login, or closes tab on the login and comes back later - they may get redirected somewhere unexpected?)